src/dev/build/tasks/os_packages/run_fpm.ts

```diff
@@ -129,6 +129,9 @@ export async function runFpm(
     // copy the data directory at /var/lib/kibana
     `${resolveWithTrailingSlash(fromBuild('data'))}=/var/lib/kibana/`,
 
+    // copy the logs directory at /var/log/kibana
```

Comment:
Can we add an     '--exclude', `usr/share/kibana/logs`, a few lines up so we don't copy the .gitempty/folder over?

Reply:
Done 👍